lib/onebox/engine/cloudapp_onebox.rb

```diff
@@ -22,33 +22,27 @@ def to_html
       private
 
       def link_html(og)
-        escaped_url = ::Onebox::Helpers.normalize_url_for_output(og[:url])
-
         <<-HTML
-            <a href='#{escaped_url}' target='_blank'>
-              #{og[:title]}
+            <a href='#{og.url}' target='_blank'>
+              #{og.title}
             </a>
           HTML
       end
 
       def video_html(og)
-        src = og[:url]
-        title = og[:title]
-        direct_src = ::Onebox::Helpers.normalize_url_for_output("#{src}/#{title}")
+        direct_src = ::Onebox::Helpers.normalize_url_for_output("#{og.url}/#{og.title}")
```

Comment:
Should this use use a version of `url` that isn't normalized yet in order to prevent double normalization?